cmd/executable_test.go

```diff
@@ -0,0 +1,24 @@
+package cmd
+
+import "testing"
+import "context"
+
+// TestExecutableExists validates binary exists
+func TestExecutableExists(t *testing.T) {
```

Comment:
We should also have a test that tests what happens when the checker does not find a binary.